src/e2e-browser/pageModels/broker.ts

```diff
@@ -23,40 +23,20 @@ import { t, ClientFunction, Selector } from "testcafe";
 import { screen } from "@testing-library/testcafe";
 
 export class BrokerPage {
-  accessOpenIdCheckbox: Selector;
-  accessOfflineCheckbox: Selector;
-  accessWebIdCheckbox: Selector;
-  rememberForeverRadioButton: Selector;
-  rememberOneHourRadioButton: Selector;
-  rememberNotRadioButton: Selector;
   authoriseButton: Selector;
   denyButton: Selector;
 
   constructor() {
-    this.accessOpenIdCheckbox = screen.getByLabelText(
-      "log in using your identity"
-    );
-    this.accessOfflineCheckbox = screen.getByLabelText("offline access");
-    this.accessWebIdCheckbox = screen.getByLabelText("solid webid");
-    this.rememberForeverRadioButton = screen.getByLabelText(
-      "remember this decision until I revoke it"
-    );
-    this.rememberOneHourRadioButton = screen.getByLabelText(
-      "remember this decision for one hour"
-    );
-    this.rememberNotRadioButton = screen.getByLabelText(
-      "prompt me again next time"
-    );
-    this.authoriseButton = screen.getByText("Authorize");
+    this.authoriseButton = screen.getByText("Approve");
     this.denyButton = screen.getByText("Deny");
   }
 
   async authoriseOnce() {
     await onAuthorisePage();
-    await t.click(this.rememberNotRadioButton).click(this.authoriseButton);
+    await t.click(this.authoriseButton);
```

Comment:
Probably needs a general search-and-replace (e.g., line 34 too!)
```suggestion
    await t.click(this.authorizeButton);
```

Reply:
The codebase internally has mostly used British English spelling for pretty much everything - not sure if it's feasible to change it all in one go (don't think I'd even be able to - I wouldn't know which are which), but let's not change them on a case-by-case basis.